rpc/src/v1/traits/net.rs

```diff
@@ -30,12 +30,20 @@ pub trait Net: Sized + Send + Sync + 'static {
 	/// Otherwise false.
 	fn is_listening(&self, _: Params) -> Result<Value, Error>;
 
+	/// Start the network.
+	fn start_network(&self, _: Params) -> Result<Value, Error>;
+
+	/// Stop the network.
+	fn stop_network(&self, _: Params) -> Result<Value, Error>;
+
 	/// Should be used to convert object to io delegate.
 	fn to_delegate(self) -> IoDelegate<Self> {
 		let mut delegate = IoDelegate::new(Arc::new(self));
 		delegate.add_method("net_version", Net::version);
 		delegate.add_method("net_peerCount", Net::peer_count);
 		delegate.add_method("net_listening", Net::is_listening);
+		delegate.add_method("net_start", Net::start_network);
```

Comment:
It will be possible to enable/disable network for a potential attacker (browsing malicious website with Flash enabled) are we ok with that?

If not we should put the API to separate trait, in future it could require confirmation using Trusted Signer.